playbooks/infra/deploy-bm-hypervisor.yml

```diff
@@ -0,0 +1,227 @@
+        state: present
+        disable_gpg_check: true
+
+    - name: Activate OS
+      ansible.builtin.command:
+        "{{ activate_system_cmd }}"
+      changed_when: false
+
+    - name: Set paswordless sudo
+      ansible.builtin.lineinfile:
+        path: /etc/sudoers.d/{{ ansible_user }}
+        line: "{{ ansible_user }} ALL=(ALL) NOPASSWD: ALL"
+        mode: "0640"
+        create: true
+
+    - name: Configure network connection baremetal
+      community.general.nmcli:
+        type: bridge
+        conn_name: bridge-baremetal
+        method4: manual
+        method6: disabled
+        state: present
+        stp: false
+        ifname: baremetal
+        autoconnect: true
+        ip4: "{{ hostvars[inventory_hostname]['ip'] }}/24"
+        gw4: "{{ hostvars[inventory_hostname]['gw'] }}"
+        dns4:
+          - "{{ hostvars[inventory_hostname]['dns'] }}"
+
+    - name: Set up network connection bridge-slave
+      community.general.nmcli:
+        type: ethernet
+        slave_type: bridge
+        ifname: "{{ hostvars[inventory_hostname]['interface_name'] }}"
+        master: baremetal
+        method4: disabled
+        conn_name: "{{ hostvars[inventory_hostname]['interface_name'] }}"
+        state: present
+        autoconnect: true
+
+    - name: Reload NetworkManager connections
+      ansible.builtin.shell: |
+        nmcli con down eno8303 && nmcli con up eno8303 && nmcli con up bridge-baremetal && nmcli con up eno8303
+      changed_when: true
+
+    - name: Gather facts
+      ansible.builtin.gather_facts:
+
+    - name: Configure secondary interface bridges
+      when: item.value.ifname in ansible_facts.interfaces
+      loop: "{{ seconday_networks | from_yaml | dict2items }}"
+      community.general.nmcli:
+        type: bridge
+        conn_name: "{{ item.key }}"
+        method4: manual
+        method6: disabled
+        state: present
+        stp: false
+        ifname: "{{ item.key }}"
+        autoconnect: true
+        ip4: "{{ item.value.ipv4 }}"
+
+    - name: Configure vlan interfaces
+      when: item.value.ifname in ansible_facts.interfaces
+      loop: "{{ seconday_networks | from_yaml | dict2items }}"
+      community.general.nmcli:
+        type: vlan
+        conn_name: "vlan{{ item.value.vlan }}"
+        state: present
+        ifname: "vlan{{ item.value.vlan }}"
+        autoconnect: true
+        slave_type: bridge
+        vlanid: "{{ item.value.vlan }}"
+        master: "{{ item.key }}"
+        vlandev: "{{ item.value.ifname }}"
+
+    - name: Install virtualization packages
+      ansible.builtin.dnf:
+        name:
+          - qemu-kvm
+          - libvirt
+          - virt-install
+          - virt-viewer
+          - libguestfs-tools-c
+        state: present
+
+    - name: Add the user to libvirt group
+      ansible.builtin.user:
+        name: "{{ ansible_user }}"
+        groups: libvirt
+        append: true
+
+    - name: Libvir insert lines for non-root user
```

Comment:
- Perhaps the task name should be - "Allow {{ ansible_user }} to manage VMs"? 
- If you prefer to leave the task's name as is - there is a slight typo in the `Libvir` .
-  Shouldn't the group be "libvirt"?

Reply:
Group should be the user's group. The rest was fixed.